### PR TITLE
add notes for n26114

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

File: _posts/2022-10-05-#26114.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+	- The `ThreadOpenConnections()` thread processes and sends messages to this node's peers.
+
+	- The `ThreadDNSAddressSeed` thread makes connections to DNS seeds.
+
+	- This PR also moves handling of fixed seeds from `ThreadOpenConnections` to
+	  `ThreadDNSAddressSeed`, and renames the latter to `ThreadAddressSeed`. Note this is
+	more involved than refactoring some code from one function to another; moving logic from one
+	thread to another could mean that data structures previously accessed by one thread are now
+	shared between multiple.
```
Comment (stickies-v) [inline _posts/2022-10-05-#26114.md:57]: Would it be useful to mention that we also need to be more careful about order of operations across threads and data races?

File: _posts/2022-10-05-#26114.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Questions
+
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
+
+1. Under what circumstances do we connect to the fixed seeds?
+
+1. What observable behavior change does this PR introduce? What addresses do we add to `AddrMan`,
```
Comment (stickies-v) [inline _posts/2022-10-05-#26114.md:65]:
```suggestion
1. What observable behavior change does this PR introduce? What kind of addresses do we add to `AddrMan`,
```

**Review by stickies-v — APPROVED** [review]
Looks great, thank you glozow! Two minor comments but good to go for me.

**Comment by glozow** [comment]
Thanks @stickies-v!

**Pull request merged**